posthog/queries/trends/formula.py

```diff
@@ -97,6 +97,9 @@ def _run_formula_query(self, filter: Filter, team: Team):
             response = []
             for item in result:
                 additional_values: Dict[str, Any] = {"label": self._label(filter, item)}
+                if filter.breakdown:
+                    additional_values["breakdown_value"] = additional_values["label"]
```

Comment:
This is not correct, we should just have the formula here (e.g. "A + B") instead of a random breakdown value

| Before | After |
|--------|--------|
| <img width="371" alt="Screenshot 2023-06-01 at 16 24 15" src="https://github.com/PostHog/posthog/assets/4550621/ef5709c7-d4d2-424f-abd1-e9cde2c56054"> | <img width="508" alt="Screenshot 2023-06-01 at 16 25 32" src="https://github.com/PostHog/posthog/assets/4550621/16d116da-b77b-4503-a732-3caec2c5253e"> |

Reply:
In formula mode it's also only possible to have one output, so I think the existing compact pattern was very fitting. The only wrong thing about it is including A/B/C/etc., but if we can hide just that, this should be solved

Reply:
Hmm... Back to the drawing board then. The problem reported was with the "Bar Value" chart, and I failed to double check this one...

Reply:
I took over this as it came up again in ZEN-3400